packages/react-langgraph/src/index.ts

```
@@ -12,11 +12,19 @@ export {
   type LangGraphSendMessageConfig,
   type LangGraphStreamCallback,
 } from "./useLangGraphMessages";
-export { convertLangchainMessages } from "./convertLangchainMessages";
+export { convertLangChainMessages as convertLangChainMessages } from "./convertLangChainMessages";
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)